odoo/tests/common.py

```diff
@@ -37,7 +37,7 @@
 from functools import lru_cache, partial
 from itertools import zip_longest as izip_longest
 from passlib.context import CryptContext
-from typing import Optional, Iterable
+from typing import Optional, Iterable, Self
```

Comment:
Doesn't work with python3.10, which is the minimal-supported version:

https://github.com/odoo/odoo/blob/eeddbd9ab2117545ad052cf376f6ec72c92299e9/odoo/release.py#L39